deluge/ui/console/modes/input_popup.py

```diff
@@ -107,11 +107,10 @@ def __init__(self, parent, message, name, child, checked=False, additional_forma
         self.name = name
         self.checked = checked
         self.msglen = len(self.chkd_inact) + 1
-        self.child = child
```

Comment:
by fixed i mean ive made changes to this file in this case just a revert of this line